assets/charts/components/dex/templates/05-ingress.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: dex
+  namespace: dex
```

Comment:
we shouldn't set namespace field, helm will ensure resources are created in correct namespace.